README.md

```diff
@@ -1,121 +1,103 @@
-----------------------------------------
+```bash
+sassdoc <src> --theme herman
+```
 
-If you use an `@example` annotation with the `scss` or `njk` languages,
-Herman will display both the source code of the example
-and its rendered output.
 
-This makes it possible to show examples of markup patterns
-that make use of styles in your Sass.
+### SassDoc Comments
 
-For example, this:
+Currently,
+all SassDoc/Herman annotations are written as Sass comments
+starting with `///` to differentiate documentation
+from other developer comments (`//`).
 
-	// Test Macro
-	// ----------
-	// This is a test.
-	/// @example njk - Basic usage:
-	///  {% import 'macros.j2' as macros %}
-	///  {{ macros.mymacro(1, 2) }}
-	[data-mymacro] {}
+```scss
+// This comment will be ignored by Herman
+/// This comment will be rendered in the documentation
+```
 
-will render the `mymacro` macro from the file `macros.j2`
-(which happens to use the `[data-mymacro]` attribute).
+Annotation comments can be free-floating,
+or attached to a particular Sass/CSS object –
+such as a variable, mixin, function, or selector block.
 
-In order for this to work for scss, you must include an `includepaths` key in
-your sassdoc configuration `herman.sass` option object. It should be an array
-of places to look for Sass includes, like
-`includepaths: [ path.join(__dirname, 'static/sass') ]`. Further, all of
-the Sass examples will have to be complete and valid. This may require
-including something like `@import 'config/manifest';` at the top of each. If
-you include an `includes` array in your sassdoc configuration `herman.sass`
-option object, those files (relative to the `includepaths`) will always be
-`@import`-ed for `@example scss`. *Note: Included Sass files should not contain
-any CSS output; any output will be displayed along with the @example.*
+```scss
+/// this is a free-floating comment
 
-In order for this to work for nunjucks,
-you must also either specify a `templatepath`
-(the path where nunjucks will look to import templates),
-or a `nunjucksEnv` (a custom nunjucks environment —
-this is particularly useful if your macros contain custom filters)
-in your sassdoc configuration `herman` option object.
+/// this comment is attached to the following mixin code-block
+@mixin sample-object { … }
+```
 
-Any html (or html-compiled) examples
-will also be rendered in iframes.
 
+### Herman Annotations
 
-Development
------------
+In addition to the core SassDoc annotations,
+our `@icons` annotation allows you to
+display SVG icons from a given folder,
+and we extend the core `@example` annotation
+to display compiled Sass/Nunjucks output
+and render sample components.
+We also provide a `@preview` annotation
+for displaying color-palettes, text and spacing sizes,
+and modular ratios, and a `@font` annotation
+for displaying font-specimens.
 
-To install the necessary Node dependencies, run ``yarn``.
+[See the full documentation for details »][docs]
```

Comment:
I think this should become a link to the annotation-specific docs (#183), and we should add a (recursive) link to these docs at the very top.

Reply:
Sounds good! We can do that in the new issue.